CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
-# Changelog
+Changelog
+=========
+
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [2.0.0] - 2021-12-09
+
+Change the public api and refactor to classes
+
+### Breaking
+- The public api has changed, a lot
+- previous default export function is gone, an exported parse function is added instead with a different signature
+- Minimum node version is now 14 since URL behaves' in earlier versions
 
-## [Unreleased]
+### Changes
+- More descriptive expression error messages
```

Comment:
Use the instructions from [keep a changelog](https://keepachangelog.com/en/1.0.0/) format documentation

```md
### Changed
- Public API
- Minimum node version is now 14 since URL behaves' in earlier versions

### Removed
- Previous default export function is gone, replaced by `parse`

### Fixed
- More descriptive expression error messages
```